backend/stakepoold/rpc/rpcserver/server.go

```diff
@@ -155,3 +155,26 @@ func (s *stakepooldServer) ImportScript(ctx context.Context, req *pb.ImportScrip
 		HeightImported: heightImported,
 	}, nil
 }
+
+func (s *stakepooldServer) StakePoolUserInfo(ctx context.Context, req *pb.StakePoolUserInfoRequest) (*pb.StakePoolUserInfoResponse, error) {
+	response, err := s.appContext.StakePoolUserInfo(req.MultiSigAddress)
+	if err != nil {
+		return nil, err
+	}
+
+	tickets := make([]*pb.StakePoolUserTicket, 0)
```

Comment:
```go
tickets := make([]*pb.StakePoolUserTicket, 0)
```

Reply:
Your comment is the same as the line of code.  I have assumed a copy/paste error, and assumed you wanted me to set the length of the created slice. Updated to

```
tickets := make([]*pb.StakePoolUserTicket, len(response.Tickets))
```

Reply:
I think he meant `tickets := make([]*pb.StakePoolUserTicket, 0, len(response.Tickets))`

Reply:
oops, i pasted it but forgot to edit it :)

Reply:
👍 
```go
tickets := make([]*pb.StakePoolUserTicket, 0, len(response.Tickets))
```